README.md

```diff
@@ -21,6 +18,65 @@ python -m pip install .
 
 ## Usage
 
+### Set up Data Version Control (DVC)
+
+This is an optional step, but is highly recommended.
+
+`dmd-era5` uses [DVC](https://dvc.org/) to manage data versioning. DVC allows to keep multiple versions of a dataset with the same file name, and to set up a remote storage location to share datasets between project collaborators. Most DVC steps are automated, but you will need to set up DVC for the first time. DVC will have been installed as a dependency when you installed this package. DVC works alongside Git to track changes to data files, so it is a good idea to work in a new branch off the main branch when using `dmd-era5` with DVC:
+
+```bash
+git checkout -b my-working-branch
+```
```

Comment:
The package has the look and feel of a library that provides some stand-alone functionality, but really seems to be based around a workflow of running from within its own repository - but is not obviously structured as such.

Would be good to make sure this is clear in the usage - but perhaps it could be made to run outside its own repo without too much effort?

Needing to create a new branch just to run the workflow hints at this I think.  It would be nice to be able to create a separate directory for handling and downloading the data and versioning the configs of the runs I want to do, without having to care about the package's own version history.

Reply:
Yes, good point. The fact that it currently needs to be run from within its own repo needs to be added to the docs. And agreed, eventually would be nice to be made to run outside its own repo. Nevertheless, you still need to use Git to track your data versions, regardless of whether you are using the package's repo or a different directory.

Reply:
> Nevertheless, you still need to use Git to track your data versions

True, and agree mentioning this in the usage is reasonable.

The rest of my comment doesn't really apply to the selected lines, and is more of a general observation. It isn't something introduced by this PR, so can discuss on the issue you just opened and not block the merge.